From 753a3c5e0d98ae2714daea2ad06e1a815462f2c0 Mon Sep 17 00:00:00 2001
From: Evgueni Souleimanov <esoule@100500.ca>
Date: Wed, 6 Jun 2018 19:15:52 -0400
Subject: [PATCH 2/2] Add missing use statements in bflt handler

Signed-off-by: Evgueni Souleimanov <esoule@100500.ca>
---
 lib/File/StripNondeterminism/handlers/bflt.pm | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/File/StripNondeterminism/handlers/bflt.pm b/lib/File/StripNondeterminism/handlers/bflt.pm
index 637e495919fa..5c0b43c39c3b 100644
--- a/lib/File/StripNondeterminism/handlers/bflt.pm
+++ b/lib/File/StripNondeterminism/handlers/bflt.pm
@@ -20,6 +20,8 @@ package File::StripNondeterminism::handlers::bflt;
 
 use strict;
 use warnings;
+
+use File::StripNondeterminism;
 use Fcntl q/SEEK_SET/;
 
 use constant bFLT => 0x62464C54;
-- 
2.17.0

